pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml

```diff
@@ -1,5 +1,5 @@
 <!--
-Note: the first 4 bits is genre and last is sub genre
+Note: the first 4-bits is genre and last is sub genre
```

Comment:
This is not a typo, the dash should not be added.

Reply:
Okay Thank You